packages/fxa-settings/src/models/verification/account-recovery-key-info.ts

```diff
@@ -0,0 +1,47 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { bind, ContextValidation as V, ModelContextProvider, ModelContext, validateContext, ContextValidationErrors } from '../../lib/context';
+
+export class AccountRecoveryKeyInfo implements ModelContextProvider
```

Comment:
@millsoper  Here is a much simpler example of using bind decorators attach to an underlying context. This model is much simpler than reliers.